packages/lib/src/components/AmazonPay/AmazonPay.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../style';
+
+.adyen-checkout__amazonpay__button {
+    margin: auto;
+}
+
+.adyen-checkout__button--ghost {
```

Comment:
Should this belong to `Button.scss`? Or is really specifc to AmazonPay?

Reply:
These styles are only for Amazon Pay. I'll add a selector to limit the scope of the style. Good catch, cheers!